configure.ac

```diff
@@ -420,7 +421,7 @@ AC_MSG_CHECKING([whether to build tests])
 
 # Tests
 
-NL_ENABLE_TESTS([yes])
```

Comment:
Why did we get rid of this?

Reply:
Mistakenly took it out, added it back in the latest set of commits.